src/main/scala/bio/sequence/codonsequence.scala

```diff
@@ -65,17 +65,15 @@ package Protein {
         seq.grouped(3).toSeq
 
       // Amino acids and nucleotides
-      val aas = ToSequence(str).translate() // IUPAC Sequence
```

Comment:
Why remove this comment? (And similar below)
Does it add no new information in the context?

Reply:
I believe in code being self-explanatory, that's why I am against commenting. 
In this case, I re-checked the code now and I might mistaken them for commented code.
I will roll-back these changes.

Reply:
Yep, I agree with you. In this case I think it was ambiguous, so either way would probably have been fine in the end.